TestON/tests/FuncIntent/Dependency/FuncIntentFunction.py

```diff
@@ -543,6 +550,7 @@ def singleToMultiIntent( main,
                                             tcpSrc="",
                                             tcpDst="" ) )
 
+    time.sleep( 10 )
```

Comment:
Okay~ I think I was waiting for the flows to go through out the instances.

On Fri, May 29, 2015 at 12:25 PM, Jon Hall <notifications@github.com> wrote:

> In TestON/tests/FuncIntent/Dependency/FuncIntentFunction.py
> <https://github.com/opennetworkinglab/ONLabTest/pull/123#discussion_r31356715>
> :
>
> > @@ -543,6 +550,7 @@ def singleToMultiIntent( main,
> >                                              tcpSrc="",
> >                                              tcpDst="" ) )
> >
> > +    time.sleep( 10 )
>
> can you document what this is for?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opennetworkinglab/ONLabTest/pull/123/files#r31356715>.
>